addons/web/static/src/views/fields/properties/properties_field.js

```diff
@@ -81,6 +82,88 @@ export class PropertiesField extends Component {
         );
 
         useEffect(() => this._movePopoverIfNeeded());
+
+        // on mono-column layout, allow to move before a separator to make the usage more fluid
+        const additionnalSelector =
+            this.columns === 1 ? ", .o_property_group .o_field_property_group_label" : "";
+        // sort properties
+        useSortable({
+            enable: () => true,
+            ref: this.propertiesRef,
+            handle: ".o_field_property_label .oi-draggable",
+            elements: ".o_property_field" + additionnalSelector,
+            groups: ".o_property_group",
+            connectGroups: true,
+            cursor: "grabbing",
+            onDragStart: ({ element, group }) => {
+                this.propertiesRef.el.classList.add("o_property_dragging");
```

Comment:
Idk if it's better or not, but for knowledge's sidebar I used something like 
```js
onDragStart: () => this.state.dragging = true,
onDragEnd: () => this.state.dragging = false,
 ```
 And added the classes in the template based on this state variable.